17:13:01,503 root DEBUG [Errno 2] No such file or directory: 'bp.csv'
17:24:03,212 root DEBUG list index out of range
17:24:03,213 root DEBUG name 'view' is not defined
17:24:03,214 root DEBUG name 'view' is not defined
17:42:58,186 root DEBUG list index out of range
17:43:01,700 root DEBUG 'in <string>' requires string as left operand, not NoneType
17:47:07,452 root DEBUG list index out of range
17:47:07,453 root DEBUG 'NoneType' object is not iterable
17:47:07,453 root DEBUG 'NoneType' object is not iterable
17:47:29,868 root DEBUG [Errno 2] No such file or directory: 'bp.csv'
17:47:29,868 root DEBUG 'NoneType' object has no attribute 'replace'
17:47:45,563 root DEBUG [Errno 2] No such file or directory: 'bp.csv'
17:47:45,571 root DEBUG 'NoneType' object has no attribute 'replace'
17:48:17,845 root DEBUG [Errno 2] No such file or directory: 'bp.csv'
17:48:39,955 root DEBUG list index out of range
17:48:39,955 root DEBUG 'NoneType' object is not iterable
17:48:39,956 root DEBUG 'NoneType' object is not iterable
17:56:29,872 root DEBUG [Errno 2] No such file or directory: 'bp.csv'
17:56:51,720 root DEBUG list index out of range
17:59:43,784 root DEBUG [Errno 2] No such file or directory: 'bp.csv'
18:00:11,288 root DEBUG list index out of range
18:00:11,289 root DEBUG 'NoneType' object is not iterable
18:01:02,87 root DEBUG [Errno 2] No such file or directory: 'bp.csv'
18:01:02,87 root DEBUG 'NoneType' object has no attribute 'replace'
18:01:02,87 root DEBUG 'NoneType' object is not iterable
18:04:24,914 root DEBUG list index out of range
18:04:44,338 root DEBUG list index out of range
18:06:08,466 root DEBUG list index out of range
18:08:01,105 root DEBUG 'list' object has no attribute 'apend'
18:14:04,924 root DEBUG list.append() takes exactly one argument (0 given)
18:23:46,604 root DEBUG [Errno 2] No such file or directory: 'bp.csv'
19:26:39,918 root DEBUG string index out of range
19:27:06,286 root DEBUG string index out of range
19:33:36,466 root DEBUG [Errno 2] No such file or directory: 'bp.csv'
19:34:51,975 root DEBUG [Errno 2] No such file or directory: 'bp.csv'
19:34:54,991 root DEBUG [Errno 2] No such file or directory: 'bp.csv'
19:34:54,991 root DEBUG 'NoneType' object has no attribute 'replace'
19:34:56,824 root DEBUG [Errno 2] No such file or directory: 'bp.csv'
19:34:56,824 root DEBUG 'NoneType' object has no attribute 'replace'
19:34:57,607 root DEBUG [Errno 2] No such file or directory: 'bp.csv'
19:34:57,607 root DEBUG 'NoneType' object has no attribute 'replace'
19:34:58,356 root DEBUG [Errno 2] No such file or directory: 'bp.csv'
19:34:58,356 root DEBUG 'NoneType' object has no attribute 'replace'
19:34:58,969 root DEBUG [Errno 2] No such file or directory: 'bp.csv'
19:34:58,969 root DEBUG 'NoneType' object has no attribute 'replace'
19:36:47,130 root DEBUG [Errno 2] No such file or directory: 'bp.csv'

09:45:45,208 root DEBUG [Errno 2] No such file or directory: 'log_2022/12/18.log'
09:46:47,171 root DEBUG [Errno 2] No such file or directory: 'log_2022/12/18.log'
09:46:51,145 root DEBUG [Errno 2] No such file or directory: 'log_2022/12/18.log'
09:47:53,225 root DEBUG [Errno 2] No such file or directory: './logs/log_2022/12/18.log'
09:48:17,67 root DEBUG [Errno 2] No such file or directory: './log_dir/log_2022/12/18.log'
09:48:48,856 root DEBUG 'int' object is not callable
09:49:02,98 root DEBUG 'int' object is not callable
09:49:02,576 root DEBUG 'int' object is not callable
09:49:02,804 root DEBUG 'int' object is not callable
09:49:02,950 root DEBUG 'int' object is not callable
09:49:03,122 root DEBUG 'int' object is not callable
09:49:03,288 root DEBUG 'int' object is not callable
09:49:03,444 root DEBUG 'int' object is not callable
09:49:03,624 root DEBUG 'int' object is not callable
09:49:03,805 root DEBUG 'int' object is not callable
09:49:03,969 root DEBUG 'int' object is not callable
09:49:04,135 root DEBUG 'int' object is not callable
09:49:04,304 root DEBUG 'int' object is not callable
09:49:45,143 root DEBUG 'int' object is not callable
09:49:45,813 root DEBUG 'int' object is not callable
09:49:45,977 root DEBUG 'int' object is not callable
09:49:46,150 root DEBUG 'int' object is not callable
09:49:46,320 root DEBUG 'int' object is not callable
09:49:46,471 root DEBUG 'int' object is not callable
09:51:11,428 root DEBUG 'int' object is not callable
2022-12-18 08:56:56,861 root DEBUG invalid literal for int() with base 10: ' '
2022-12-18 08:56:59,119 root DEBUG invalid literal for int() with base 10: ' '
2022-12-18 09:17:44,359 root DEBUG [view.py:73 -         input_choose() ] invalid literal for int() with base 10: ' '
2022-12-18 09:48:32,118 root DEBUG [view.py:73 -         input_choose() ] invalid literal for int() with base 10: '���'
2022-12-18 09:48:34,92 root DEBUG [view.py:73 -         input_choose() ] invalid literal for int() with base 10: '���'
2022-12-18 10:32:08,664 root DEBUG [view.py:73 -         input_choose() ] invalid literal for int() with base 10: ' �����'
2022-12-20 09:52:02,284 root DEBUG [controller.py:78 - print_array_data_from() ] list index out of range
2022-12-20 09:52:39,759 root DEBUG [controller.py:78 - print_array_data_from() ] list index out of range
2022-12-20 09:52:49,608 root DEBUG [controller.py:78 - print_array_data_from() ] list index out of range
2022-12-20 09:52:50,947 root DEBUG [controller.py:78 - print_array_data_from() ] list index out of range
2022-12-20 09:53:03,971 root DEBUG [controller.py:78 - print_array_data_from() ] list index out of range
2022-12-20 09:54:19,948 root DEBUG [controller.py:78 - print_array_data_from() ] list index out of range
2022-12-20 21:06:07,468 root DEBUG [controller.py:88 - print_array_data_from() ] local variable 'item' referenced before assignment
2022-12-20 21:26:06,808 root DEBUG [view.py:66 -         input_choose() ] invalid literal for int() with base 10: '& "C:/Program Files/Python310/python.exe" c:/Users/me/PhonesDir/TelephoneDirectory/logger_file.py'
2022-12-20 21:26:31,382 root DEBUG [controller.py:88 - print_array_data_from() ] Invalid format string
2022-12-20 21:26:31,383 root DEBUG [controller.py:44 -               search() ] Invalid format string
2022-12-20 21:26:38,301 root DEBUG [controller.py:63 -          add_contact() ] Invalid format string
2022-12-20 21:26:39,783 root DEBUG [view.py:66 -         input_choose() ] invalid literal for int() with base 10: '\\'
2022-12-20 21:30:27,755 root DEBUG [view.py:66 -         input_choose() ] invalid literal for int() with base 10: '& "C:/Program Files/Python310/python.exe" c:/Users/me/PhonesDir/TelephoneDirectory/controller.py'
2022-12-20 21:30:31,899 root DEBUG [view.py:66 -         input_choose() ] invalid literal for int() with base 10: '& "C:/Program Files/Python310/python.exe" c:/Users/me/PhonesDir/TelephoneDirectory/main.py'
2022-12-20 21:30:33,155 root DEBUG [view.py:66 -         input_choose() ] invalid literal for int() with base 10: '& "C:/Program Files/Python310/python.exe" c:/Users/me/PhonesDir/TelephoneDirectory/main.py'
2022-12-20 21:30:33,666 root DEBUG [view.py:66 -         input_choose() ] invalid literal for int() with base 10: '& "C:/Program Files/Python310/python.exe" c:/Users/me/PhonesDir/TelephoneDirectory/main.py'
2022-12-20 21:30:34,132 root DEBUG [view.py:66 -         input_choose() ] invalid literal for int() with base 10: '& "C:/Program Files/Python310/python.exe" c:/Users/me/PhonesDir/TelephoneDirectory/main.py'
2022-12-20 21:30:34,538 root DEBUG [view.py:66 -         input_choose() ] invalid literal for int() with base 10: '& "C:/Program Files/Python310/python.exe" c:/Users/me/PhonesDir/TelephoneDirectory/main.py'
2022-12-20 21:30:51,574 root DEBUG [controller.py:64 -          add_contact() ] Invalid format string
2022-12-20 21:30:58,60 root DEBUG [controller.py:89 - print_array_data_from() ] Invalid format string
2022-12-20 21:30:58,61 root DEBUG [controller.py:45 -               search() ] Invalid format string
2022-12-20 21:31:08,628 root DEBUG [controller.py:77 -       delete_contact() ] Invalid format string
2022-12-20 21:31:16,963 root DEBUG [controller.py:77 -       delete_contact() ] Invalid format string
2022-12-20 21:31:17,728 root DEBUG [view.py:66 -         input_choose() ] invalid literal for int() with base 10: ''
2022-12-20 21:32:05,375 root DEBUG [controller.py:89 - print_array_data_from() ] Invalid format string
2022-12-20 21:32:05,376 root DEBUG [controller.py:45 -               search() ] Invalid format string
2022-12-20 21:32:14,428 root DEBUG [controller.py:64 -          add_contact() ] Invalid format string
2022-12-20 21:32:17,344 root DEBUG [controller.py:89 - print_array_data_from() ] Invalid format string
2022-12-20 21:32:17,344 root DEBUG [controller.py:54 -      print_directory() ] Invalid format string
2022-12-20 21:33:23,629 root DEBUG [controller.py:89 - print_array_data_from() ] local variable 'item' referenced before assignment
2022-12-20 21:33:23,629 root DEBUG [controller.py:45 -               search() ] Invalid format string
2022-12-20 21:33:51,886 root DEBUG [controller.py:89 - print_array_data_from() ] Invalid format string
2022-12-20 21:33:51,887 root DEBUG [controller.py:45 -               search() ] Invalid format string
2022-12-20 21:33:56,797 root DEBUG [controller.py:64 -          add_contact() ] Invalid format string
2022-12-20 21:37:32,409 root DEBUG [view.py:66 -         input_choose() ] invalid literal for int() with base 10: '& "C:/Program Files/Python310/python.exe" c:/Users/me/PhonesDir/TelephoneDirectory/main.py'
2022-12-20 21:39:03,27 root DEBUG [view.py:66 -         input_choose() ] invalid literal for int() with base 10: '& "C:/Program Files/Python310/python.exe" c:/Users/me/PhonesDir/TelephoneDirectory/main.py'
